docs/news.rst

```diff
@@ -3,6 +3,57 @@
 Release notes
 =============
 
+.. _release-2.7.1:
+
+Scrapy 2.7.1 (Not relased yet)
+------------------------------
+
+New features
+~~~~~~~~~~~~
+
+-   Relaxed the restriction introduced in 2.6.2 so that the
+    ``Proxy-Authentication`` header can again be set explicitly, as long as the
+    proxy URL in the :reqmeta:`proxy` metadata has no other credentials, and
+    for as long as that proxy URL remains the same (:issue:`5626`).
```

Comment:
What do you think about mentioning https://github.com/scrapy-plugins/scrapy-zyte-smartproxy here? Would it be helpful for our users?

Reply:
Oh yeah. Like "this restores compatibility with older versions of it"?

Reply:
yep; it makes sense to call out specific version number as well

Reply:
> it makes sense to call out specific version number as well

I suspect this restores support for *all* previous versions of scrapy-zyte-smartproxy/scrapy-crawlera, though I did not confirm this.

Reply:
A good point about the lower bound; I was thinking only about mentioning a highest version number. It doesn't look like a blocker though if we don't provide the oldest version number here.

Reply:
Oh, “scrapy-zyte-smartproxy 2.1.0 and older”, yes, that makes sense!

Reply:
Done. Should we also mention 3rd-party proxy extensions here (without specific names) or is this enough?

Reply:
I think this is enough.